src/Schema/SqliteSchemaManager.php

```diff
@@ -582,7 +582,7 @@ private function getCreateTableSQL(string $table): string
 AND name = ?
 SQL
             ,
-            [$table],
+            [trim($table, '"')], // Unquote the identifier
```

Comment:
It probably should be done here: https://github.com/doctrine/dbal/blob/8ce05eda99ddcb97ea2ef5667d2759b61438d303/src/Schema/AbstractSchemaManager.php#L252-L257

The name passed to `selectTableColumns()` is "normalized" but the one passed to `_getPortableTableColumnList()` is not.

Reply:
Okay, I updated the logic. Is there any rule about where the name should be normalized and where it should be escaped?

Reply:
> Is there any rule about where the name should be normalized and where it should be escaped?

The quoted name is what we receive to the user to allow them to declare that the original case of the name should be preserved (e.g. the lower-case `users` table should be introspected even though Oracle by default upper-cases all names). The normalized name is the result of "parsing" the user-supplied input and deriving the actual value from it.

Ideally, we want to normalize the name as soon as possible and use the normalized one downstream.